Frontend/ui-common/src/Engineering/Helpers/JBridge.js

```diff
@@ -717,11 +717,14 @@ export const drawRouteV2 = function (drawRouteConfig){
   return function() {
     try{
       console.log("Inside drawRouteV2")
-      const { locations, style, routeColor, isActual, startMarker, endMarker, routeWidth, routeType, startMarkerLabel, endMarkerLabel, mapRouteConfig } = drawRouteConfig.routes.normalRoute;
+      const configs = drawRouteConfig.routeConfigs;
       if (window.JBridge.drawRouteV2){
         return window.JBridge.drawRouteV2(JSON.stringify(drawRouteConfig));
       } else {
+        for (let i = 0; i < configs.length; i++) {
```

Comment:
@aayushagarwal2001 this loop would be terminated after the first execution as we are returning it inside the loop. Please handle the loop according to the requirement.